resolwe/flow/tests/test_models.py

```diff
@@ -1267,54 +1267,6 @@ def setUp(self):
             ],
         )
 
-    def test_save_storage(self):
```

Comment:
Since the method `save_storage` no longer exists and storage files are not stored in `data.save()` anymore. 
But I should add tests that would test the new method `handle_upload_storage`...

I will prepare them after this PR tests pass.